flatpak_builder_lint/checks/modules.py

```diff
@@ -62,8 +62,6 @@ def check_module(self, module: dict) -> None:
             for source in sources:
                 if name := module.get("name"):
                     self.check_source(name, source)
-        else:
-            self.errors.add(f"module-{name}-no-sources")
```

Comment:
instead you could check if `buildsystem` is `simple`. or a more complicated logici to detect it's one of these "sdk prep" step.